lua/terrortown/events/minigame.lua

```diff
@@ -0,0 +1,48 @@
+
+if SERVER then
+    AddCSLuaFile()
+    resource.AddFile("materials/vgui/ttt/dynamic/minigames/mg_icon.vmt")
+end
+
+if CLIENT then
+    EVENT.title = "title_minigame"
+    EVENT.icon = Material("vgui/ttt/dynamic/minigames/mg_icon.vmt")
+    
+    function EVENT:GetText()
+        local mgText = {
+            {
+                string = "desc_minigame_name",
+                params = {
+                    name = self.event.name
+                },
+                translateParams = true
+            }
+        }
+
+        if #LANG.TryTranslation(self.event.desc) > 0 then
```

Comment:
Did you meant `GetRaw` or something similar? Because `TryTranslation` even returns the given input (https://github.com/TTT-2/TTT2/blob/866babe0c80d00b95aaf7d74a00bc3384fe70edf/gamemodes/terrortown/gamemode/client/cl_lang.lua#L113). Or is it just to make sure a translation is provided? Then I suggest not using `#`, because `#nil` doesn't makes as much sense if it's not defined. Just `if LANG.TryTranslation(self.event.desc) then`, maybe with TryTranslation localized

Reply:
It's checking that the minigame has a description, otherwise it doesn't include it. Some minigames have an empty string for their description and I didn't want to include those.